.eslintrc

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
You can safely rename this file to: `.eslintrc.json`.

`.estlinrc` tends to be deprecated.
ref: https://eslint.org/docs/user-guide/configuring#configuration-file-formats

Reply:
Ok, thanks :)